element114/tusari/_posts/1992-10-01-vertical-parallax-fld.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 1992-10-01 20
+title:  "Vertical parallax FLD"
+csdb: 178390
+coders:
+  - name: "Martin Piper"
+    csdb: 19341
+record: 4
+size: 320x200
+fps: 50
+tags: fld fld-parallax fld-parallax-vertical
+---
+![Vertical parallax FLD](/c64wrd/element114/tusari/vertical-parallax-fld.png)
+
+Horizontally scrolling parallax with dynamic vertical movement provided by FLD.
+
+<!--more-->
+
+The effect shows horizontally scrolling parallax in the game (level 3 part 2) with dynamic player controlled vertical movement provided by using FLD areas to optionally separate the horizontally scrolling sections. Up to four (the record) FLD areas expand and contract at different rates depending on the player's vertical position on the screen to give a simple perspective effect.
+
+
+[YouTube link](https://www.youtube.com/watch?v=vpNisgA1yXQ)
```

Comment:
Please create a link which includes the timestamp as well (there is checkbox for that in YouTube)

Reply:
Done :) Although it was the whole video.